nautobot/circuits/navigation.py

```diff
@@ -0,0 +1,89 @@
+from nautobot.core.apps import NavMenuAddButton, NavMenuGroup, NavMenuItem, NavMenuImportButton, NavMenuTab
+
+
+menu_tabs = (
+    NavMenuTab(
+        name="Circuits",
+        weight=500,
+        groups=(
+            NavMenuGroup(
+                name="Circuits",
+                weight=100,
+                items=(
+                    NavMenuItem(
+                        link="circuits:circuit_list",
+                        link_text="Circuits",
+                        weight=100,
+                        permissions=[
+                            "circuits.view_circuit",
+                        ],
+                        buttons=(
+                            NavMenuAddButton(
+                                link="circuits:circuit_add",
+                                permissions=[
```

Comment:
Missing `weight` values on all buttons to define their ordering - though actually I don't see logic in `register_menu_items` to actually make use of a button's `weight` in any case?